src/SignalR/clients/java/signalr/src/main/java/com/microsoft/signalr/HubConnection.java

```diff
@@ -302,7 +306,13 @@ public Completable start() {
         negotiate.flatMapCompletable(url -> {
             logger.debug("Starting HubConnection.");
             if (transport == null) {
-                transport = new WebSocketTransport(headers, httpClient);
+                switch (transportEnum){
```

Comment:
Is transport fallback going to be part of this change or will it be after this goes in?